zerver/tests/test_rocketchat_importer.py

```diff
@@ -853,7 +853,11 @@ def test_do_convert_data(self) -> None:
         rocketchat_data_dir = self.fixture_file_name("", "rocketchat_fixtures")
         output_dir = self.make_import_output_dir("rocketchat")
 
-        with self.assertLogs(level="INFO") as info_log:
+        with self.assertLogs(level="INFO") as info_log, self.settings(
+            EXTERNAL_HOST="testserver.com"
```

Comment:
Let's do `zulip.example.com`; we should try to avoid domains that might actually be something, and the DNS RFCs define the `example.com` namespace for that explicit purpose.

Reply:
@timabbott Makes sense, pushed with that change. Pushed also an extra commit to get rid of some `testserver.com` in emails and replace is similarly, in case we want that? Also the rocketchat tests have a bunch of `email.com` email domains:
```
        self.assertEqual(
            exported_user_emails,
            {
                "rocket.cat-bot@zulip.example.com",
                "priyansh3133@email.com",
                "harrypotter@email.com",
                "hermionegranger@email.com",
                "ronweasley@email.com",
                "lordvoldemort@email.com",
            },
        )
```

though changing that will require modifying `.bson` rockechat test fixture files, shall i push a commit for that change too?